src/test/java/com/thoughtworks/pacman/ui/screens/LostScreenTest.java

```diff
@@ -3,44 +3,30 @@
 import com.thoughtworks.pacman.core.Game;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
 
-import java.awt.Graphics2D;
+import org.mockito.runners.MockitoJUnitRunner;
 
 import static org.hamcrest.CoreMatchers.instanceOf;
 import static org.junit.Assert.assertThat;
-import static org.mockito.Mockito.verify;
+
```

Comment:
sildiğin 

draw_shouldDrawSplashImageAcrossWidth

testine benzer testler yaz yeni iki ekranın için.

Reply:
@cagdasgerede hocam onceden denemistim, bugun de denedim ama yapamadigim icin cikarmayi uygun gordum.